runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsDrawListener.java

```diff
@@ -67,7 +67,16 @@ private FpsDrawListener(FpsConfig config)
 	void reloadConfig()
 	{
 		lastMillis = System.currentTimeMillis();
-		targetDelay = 1000 / Math.max(1, config.maxFps());
+
```

Comment:
This can just be:
```java
int fps = config.enableFpsUnfocused() && !isFocused
  ? config.maxFpsUnfocused()
  : config.maxFps();
targetDelay = 1000 / Math.max(1, fps);
```